Core/HLE/sceRtc.cpp

```diff
@@ -459,16 +459,14 @@ static int sceRtcConvertLocalTimeToUTC(u32 tickLocalPtr,u32 tickUTCPtr)
 	{
 		u64 srcTick = Memory::Read_U64(tickLocalPtr);
 		// TODO : Let the user select his timezone / daylight saving instead of taking system param ?
-#if defined(__GLIBC__) || defined(BLACKBERRY) || defined(__SYMBIAN32__)
+#ifndef _MSC_VER
```

Comment:
"The tm_gmtoff field is derived from BSD and is a GNU library extension; it is not visible in a strict ISO C environment."

I'm not actually sure if Mac has this, I guess it does?

-[Unknown]